### DISCUSSION TITLE
Change Select item background color

### DISCUSSION THREAD

**Discussion by ellieatmerqury** [mantinedev/mantine#7034] [Q&A] [answered]
### Dependencies check up

- [X] I have verified that I use latest version of all @mantine/* packages

### What version of @mantine/* packages do you have in package.json?

7.13.3

### What package has an issue?

@mantine/core

### What framework do you use?

Vite

### In which browsers you can reproduce the issue?

Chrome

### Describe the bug

Is there a way of changing the background color of the selected item/input area of the Select component? I've tried all possibly relevant options (https://mantine.dev/core/select/?t=styles-api) but still, the color remains the default one. Seeing as all other parts can be modified, I must be missing something from the docs?

```
.mantine-Select-input{
  background-color: red;
}

.mantine-Select-section{
  background-color: red;
}

.mantine-Select-root{
  background-color: red;
}

.mantine-Select-label{
  background-color: red;
}

.mantine-Select-required{
  background-color: red;
}

.mantine-Select-description{
  background-color: red;
}
.mantine-Select-error{
  background-color: red;
}
.mantine-Select-dropdown{
  background-color: red;
}
.mantine-Select-options{
  background-color: red;
}
.mantine-Select-option{
  background-color: red;
}
.mantine-Select-empty{
  background-color: red;
}
.mantine-Select-group{
  background-color: red;
}
.mantine-Select-groupLabel{
  background-color: red;
}
```

<img width="146" alt="select_background_color" src="https://github.com/user-attachments/assets/b6ac19d2-6c8a-43bf-8426-c7ac3eebc31c">

### Self-service

- [ ] I would be willing to implement a fix for this issue

**Comment by rtivital** [comment]
See example – https://codesandbox.io/p/sandbox/mantine-react-template-forked-5v53ty

**Reply by ellieatmerqury** [reply]
that seems to make sense, yet I have tried that but nothing changes, the selected item area is still dark, as shown in the original post

**Reply by rtivital** [reply, marked as answer]
See https://help.mantine.dev/q/styles-order

**Reply by ellieatmerqury** [reply]
Ah thanks will try that

**Reply by ellieatmerqury** [reply]
In hindsight that makes sense... thanks marked as the answer!
